Makefile

```diff
@@ -48,3 +48,10 @@ compose-up:
 	docker-compose up
 
 compose-restart: compose-down codegen-compose compose-up
+
+install-kustomize:
```

Comment:
The other rules of this Makefile already assume the requirements are pre-installed. I think we can drop that. You can add the tool to https://github.com/change-metrics/monocle/blob/1b7843f651fa657c56448f3dc213780b8f6c5241/nix/default.nix#L236 (e.g. `pkgs.kustomize`)

Reply:
i'll do that :)